src/api/status/src/views/partials/sidebar.hbs

```diff
@@ -4,7 +4,7 @@
   <div class="sidenav-header">
     <i class="fas fa-times p-3 cursor-pointer text-white opacity-5 position-absolute end-0 top-0 d-none d-xl-none"
       aria-hidden="true" id="iconSidenav"></i>
-    <a class="navbar-brand m-0" href="/v1/status" target="_blank">
+    <a class="navbar-brand m-0" href="https://telescope.cdot.systems/">
```

Comment:
Since we have local, staging, and production to worry about, the `href` here should not point to just `https://telescope.cdot.systems/`.

- For local development, the URL should be ``,
- For staging, the URL should be `https://dev.api.telescope.cdot.systems/v1/status/`
- For prod, the URL should be `https://api.telescope.cdot.systems/v1/status/`

We had a PR #2948 land recently that did something similar to what we want here.

Reply:
Since this is unnecessary changes, so I don't know if I can keep going or closing this one.

Reply:
Release day is tomorrow and this doesn't seem like it's a priority for 2.7, you can probably just let it brew in PR hell and get back to it for 2.8